HomeWork/src/main/java/io/FileLogger.java

```diff
@@ -0,0 +1,47 @@
+package io;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+
+public class FileLogger {
+
+    FileLoggerConfiguration configuration = new FileLoggerConfiguration();
```

Comment:
Make a configuration private, create a constructor with the param `FileLoggerConfiguration configuration`, and assign it.